src/styles/_catalog.scss

```diff
@@ -1,22 +1,42 @@
-.catalog {
-  display: flex;
-  flex-wrap: wrap;
+$column-width: 200px;
+$column-height: 300px;
 
-  max-width: 944px;
+.catalog {
+  display: grid;
+  gap: 48px;
+  place-items: center;
```

Comment:
```suggestion
  grid-template-columns: 200px;
```

Reply:
@liliia-livandovskaya Thank you. Made all the changes in the commit, however, the layout does not work as expected without  `place-items: center`, maybe I did not get the whole point you were trying to explain, so I would be very grateful for one more tip!

Reply:
yeap, you are right `place-items` should be, I deleted it by mistake while adding `grid-template-columns`. Sorry for that

Reply:
@liliia-livandovskaya So could you, please, check it one more time?